source/nodejs/README.md

```diff
@@ -19,9 +19,23 @@ We use lerna to manage package linking and building.
 2. `npm install`
 3. `npx lerna bootstrap`
 
+## Initialize Telemetry
+
+We use Microsoft 1DS as our telemetry provider.
+
+This is 1DS documentation:
+https://msasg.visualstudio.com/Shared%20Data/_git/1DS.JavaScript?path=%2Fskus%2FanalyticsWeb%2FREADME.md&version=GBmaster&fullScreen=true
```

Comment:
Hmm need to check if it's kosher to refer to internal documentation here - we'll need to follow up. They won't work anyways since it's internal links.